torch/_dynamo/variables/builtin.py

```diff
@@ -605,25 +565,22 @@ def call_mul(self, tx, a, b):
                 items=b.items * a.as_python_constant(), mutable_local=MutableLocal()
             ).add_options(self, a, b)
         else:
-            return self._call_operator_builtin(tx, "mul", a, b)
+            return a.call_method(tx, "__mul__", [b], {})
 
     def call_len(self, tx, *args, **kwargs):
         return args[0].call_method(tx, "__len__", args[1:], kwargs)
 
     def call_add(self, tx, *args, **kwargs):
-        # Important: we want to trace operator.add(x, y) into the graph,
-        # not x.__add__(y).
-        # operator.add will automatically handle NotImplemented argument swizzling.
-        return self._call_operator_builtin(tx, "add", *args, *kwargs)
+        return args[0].call_method(tx, "__add__", args[1:], kwargs)
 
     def call_sub(self, tx, *args, **kwargs):
-        return self._call_operator_builtin(tx, "sub", *args, *kwargs)
+        return args[0].call_method(tx, "__sub__", args[1:], kwargs)
 
     def call_truediv(self, tx, *args, **kwargs):
-        return self._call_operator_builtin(tx, "truediv", *args, *kwargs)
+        return args[0].call_method(tx, "__truediv__", args[1:], kwargs)
 
     def call_floordiv(self, tx, *args, **kwargs):
-        return self._call_operator_builtin(tx, "floordiv", *args, *kwargs)
+        return args[0].call_method(tx, "__floordiv__", args[1:], kwargs)
```

Comment:
So this fix will need to get reapplied right

Reply:
Yes, I plan to do that myself after this merge stuff